### ISSUE TITLE
Permissions issue when running as non-root user on 0.8.1.2135

### ISSUE THREAD

**Issue by neis** [linuxserver/docker-lidarr#67]
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

------------------------------

## Expected Behavior
Lidarr container should be able to start cleanly as non-root user

## Current Behavior
Lidarr fails to start with EPIC Fail errors (see log output below) indicating a permissions issue with /config/config.xml or /config/lidarr.pid (varies).  The ownership of these files was checked and confirmed to be owned by 1000 when the issue occurs.  However, two other directories within the config folder are showing ownership by root:
 - /config/custom-cont-init.d/
 - /config/custom-services.d/

Recursively changing the ownership of these directories to 1000 is ineffective as the ownership is changed back to root when the container is started again which in turn causes it to fail.

The same docker-compose.yml and /config directory works if the PUID and PGID are set to 0 (root).

## Steps to Reproduce

1. In docker-compose.yml set:
 - PUID=1000
 - PGID=1000
2. Run docker-compose pull
3. Run docker-compose up -d

## Environment
**OS:**  Ubuntu 18.04.6 LTS
**CPU architecture:**  x86_64
**How docker service was installed:**  Distro repo

## Command used to create docker container (run/create/compose/screenshot) compose

>version: "2.1"
>
>services:
>  lidarr:
>    image: lscr.io/linuxserver/lidarr
>    container_name: lidarr
>    environment:
>      - PUID=1000
>      - PGID=1000
>      - TZ=America/Chicago
>    volumes:
>      - /docker/lidarr/config:/config
>      - /docker/sabnzbd/downloads/complete:/downloads
>      - /media/Music:/music
>    network_mode: bridge
>    ports:
>      - 9004:9004
>    restart: unless-stopped

## Docker logs

>[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
>[s6-init] ensuring user provided files have correct perms...exited 0.
>[fix-attrs.d] applying ownership & permissions fixes...
>[fix-attrs.d] done.
>[cont-init.d] executing container initialization scripts...
>[cont-init.d] 01-envfile: executing... 
>[cont-init.d] 01-envfile: exited 0.
>[cont-init.d] 01-migrations: executing... 
>[migrations] started
>[migrations] no migrations found
>[cont-init.d] 01-migrations: exited 0.
>[cont-init.d] 10-adduser: executing... 
>
>-------------------------------------
>          _         ()
>         | |  ___   _    __
>         | | / __| | |  /  \ 
>         | | \__ \ | | | () |
>         |_| |___/ |_|  \__/
>
>
>Brought to you by linuxserver.io
>-------------------------------------
>
>To support the app dev(s) visit:
>Lidarr: https://opencollective.com/lidarr
>
>To support LSIO projects visit:
>https://www.linuxserver.io/donate/
>-------------------------------------
>GID/UID
>-------------------------------------
>
>User uid:    1000
>User gid:    1000
>-------------------------------------
>
>[cont-init.d] 10-adduser: exited 0.
>[cont-init.d] 30-config: executing... 
>[cont-init.d] 30-config: exited 0.
>[cont-init.d] 90-custom-folders: executing... 
>[cont-init.d] 90-custom-folders: exited 0.
>[cont-init.d] 99-custom-scripts: executing... 
>[custom-init] no custom files found exiting...
>[cont-init.d] 99-custom-scripts: exited 0.
>[cont-init.d] done.
>[services.d] starting services
>[services.d] done.
>[Info] Bootstrap: Starting Lidarr - /app/lidarr/bin/Lidarr.dll - Version 0.8.1.2135 
>[Info] AppFolderInfo: Data directory is being overridden to [/config] 
>
>
>[Fatal] ConsoleApp: EPIC FAIL! 
>
>[v0.8.1.2135] NzbDrone.Common.Exceptions.LidarrStartupException: Lidarr failed to start: Lidarr does not have access to config file: /config/config.xml. Please fix permissions
   >at NzbDrone.Host.Bootstrap.Start(StartupContext startupContext, IUserAlert userAlert, Action`1 startCallback) in D:\a\1\s\src\NzbDrone.Host\Bootstrap.cs:line 60
   >at NzbDrone.Console.ConsoleApp.Main(String[] args) in D:\a\1\s\src\NzbDrone.Console\ConsoleApp.cs:line 43
>
>
>Press enter to exit...
>Non-recoverable failure, waiting for user intervention...

**Comment by Roxedus** [comment]
Neither `custom-cont-init.d` or `custom-services.d` are used by Lidarr, they are not the problem. What's the permissions for `/config` itself?

**Comment by neis** [comment]
I confirmed that everything was assigned to 1000 via `ll -R | grep root` (to verify nothing returned) and the only thing that came back was the custom-* folders.  However, I just noticed that some of the files/folders were missing read permissions.  I'm not sure how it got removed as I'm running other linuxserver.io containers and I only recently pulled updated images (didn't touch the existing bind mounts...).

Either way, I appreciate the assistance.  I tried to be sure it wasn't on my end before opening the issue so I'm sorry for wasting your time.  Keep up the good work!

**Comment by neis** [comment]
I confirmed that everything was assigned to 1000 via `ll -R | grep root` (to verify nothing returned) and the only thing that came back was the custom-* folders.  However, I just noticed that some of the files/folders were missing read permissions.  I'm not sure how it got removed as I'm running other linuxserver.io containers and I only recently pulled updated images (didn't touch the existing bind mounts...).

Either way, I appreciate the assistance.  I tried to be sure it wasn't on my end before opening the issue so I'm sorry for wasting your time.  Keep up the good work!
